src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -18,450 +18,700 @@ should be included in the pipeline documentation.
+   continuous function spanning the 2-sphere as coefficient values using a
+   spherical harmonics basis.
+
+   Number of image volumes depends on the spherical harmonic basis employed,
+   and the maximal spherical harmonic degree *l<sub>max</sub>*.
+
+1. <a name="data-amp">*Amplitudes*</a>:
+
+   4D image where data across volumes within each voxel represent
+   amplitudes of a discrete function spanning the 2-sphere.
+
+   Number of image volumes corresponds to the number of discrete directions
+   on the unit sphere along which samples for the function in each voxel
+   are provided.
+
+1. <a name="data-pdf">*Probability Distribution Functions (PDFs)*</a>:
+
+   ***TODO***
+
+1. <a name="data-param">*Parameter vectors*</a>:
+
+   4D image containing, for every image voxel, data corresponding to some
+   set of model parameters, the names and order of which are defined within
+   the [intrinsic](#paramdef-intrinsic) model parameters section.
+
+### Orientation specification
+
+| **Key name**                | **Relevant [data representations](#data-representations)**                                                                                                                                                        | **Description**                                                                                                                                                                                                                                           |
+| --------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `AntipodalSymmetry`         | [Spherical coordinates](#data-spherical), [3-vectors](#data-3vector), [spherical harmonics](#data-sh), [amplitudes](#data-amp), [probability distribution functions](#data-pdf), [parameter vectors](#data-param) | OPTIONAL. Boolean. Indicates whether orientation information should be interpreted as being antipodally symmetric. Assumed to be True if omitted.                                                                                                         |
+| `Directions`                | [Amplitudes](#data-amp)                                                                                                                                                                                           | REQUIRED. List. Data are either [spherical coordinates (directions only)](#data-spherical) or [3-vectors](#data-3vector) with unit norm. Defines the dense directional basis set on which samples of a spherical function within each voxel are provided. |

```

Comment:
Possibly. In part I went with `Directions` because *MRtrix3* already supports this, and at least some commands are already compatible with either representation. Then again, the convention for spherical angles in *MRtrix3* is incompatible with what I've put into the spec, which is instead based on ISO. So maybe using different keys would actually be preferable.

Reply:
What is the status of this question?

Reply:
I think I'd like to switch to having two separate keys. Though personally my instinct was `DirectionsCartesian` and `DirectionsSpherical`. Does this clash too much with the "`3-vectors`" data representation?

Reply:
Follow up in #18.